lib/json/__tests__/output-json.test.js

```diff
@@ -1,49 +1,53 @@
-var assert = require('assert')
-var fs = require('fs')
-var os = require('os')
-var path = require('path')
-var fse = require(process.cwd())
+'use strict'
+
+const fs = require('fs')
+const os = require('os')
+const fse = require(process.cwd())
+const path = require('path')
+const assert = require('assert')
+const outputJson = require('../output-json')
 
 /* global beforeEach, describe, it */
 
-describe('json', function () {
-  var TEST_DIR
+describe('json', () => {
+  let TEST_DIR
 
-  beforeEach(function (done) {
+  beforeEach(done => {
     TEST_DIR = path.join(os.tmpdir(), 'fs-extra')
     fse.emptyDir(TEST_DIR, done)
   })
 
-  describe('+ outputJson(file, data)', function () {
-    var outputJson = require('../output-json')
-
-    it('should write the file regardless of whether the directory exists or not', function (done) {
-      var file = path.join(TEST_DIR, 'this-dir', 'prob-does-not', 'exist', 'file.json')
+  describe('+ outputJson(file, data)', () => {
+    it('should write the file regardless of whether the directory exists or not', done => {
+      const file = path.join(TEST_DIR, 'this-dir', 'prob-does-not', 'exist', 'file.json')
       assert(!fs.existsSync(file))
 
-      var data = {name: 'JP'}
-      outputJson(file, data, function (err) {
+      const data = {name: 'JP'}
+      outputJson(file, data, err => {
         if (err) return done(err)
 
         assert(fs.existsSync(file))
-        var newData = JSON.parse(fs.readFileSync(file, 'utf8'))
+        const newData = JSON.parse(fs.readFileSync(file, 'utf8'))
 
         assert.equal(data.name, newData.name)
         done()
       })
     })
 
-    describe('> when an option is passed, like JSON replacer', function () {
-      it('should pass the option along to jsonfile module', function (done) {
-        var file = path.join(TEST_DIR, 'this-dir', 'does-not', 'exist', 'really', 'file.json')
+    describe('> when an option is passed, like JSON replacer', () => {
+      it('should pass the option along to jsonfile module', done => {
+        const file = path.join(TEST_DIR, 'this-dir', 'does-not', 'exist', 'really', 'file.json')
         assert(!fs.existsSync(file))
 
-        var replacer = function (k, v) { if (v === 'JP') return 'Jon Paul'; else return v }
-        var data = {name: 'JP'}
+        const replacer = (k, v) => {
+          if (v === 'JP') return 'Jon Paul'
+          return v
+        }
```

Comment:
Could be shortened to:

    const replacer = (k, v) => v === 'JP' ? 'Jon Paul' : v

It's up to you if you want to shorten this or not.